Fix misleading text in "Hover, focus, and other states" page (#2285)
tailwindlabs/tailwindcss.com: src/docs/hover-focus-and-other-states.mdx

</Figure>

It's important to note that overriding a style with a utility directly on the child itself won't work since children rules are generated after the regular ones and they have the same specificity:

> <TipBad>{<>Won't work, children can't override styles defined on their parent.</>}</TipBad>

Reviewer: `their own styling` is referencing the fact that `*` places the style on the element itself and not on the parent element. I can see how this is confusing though. 🤔

Reviewer: ```suggestion
<TipBad>{<>Won't work, children can't override styles given to them by the parent.</>}</TipBad>
```
